extensions/health/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@loopback/extension-health",
+  "version": "1.0.0",
```

Comment:
Let's start with version `0.1.0` to allow us to do few more iterations (possibly breaking things along the way) before we call the module stable and publish `1.0.0`.